streaming-binlog/src/main/java/org/openmrs/analytics/DbeziumArgs.java

```diff
@@ -0,0 +1,69 @@
+// Copyright 2020 Google LLC
```

Comment:
Please fix the typo ("Debezium") in the file name.

Reply:
fixed